requirements.txt

```diff
@@ -1,4 +1,5 @@
-kubernetes==30.1.0 # last version with DiscoveryV1beta1Api
+lightkube==0.15.2
 ops==2.14.0
 jsonschema==4.22.0
 cryptography==42.0.8
+pydantic==1.10.15
```

Comment:
Why using pydantic 1 ? We should go for pydantic 2

Reply:
PR for this : https://github.com/canonical/gateway-api-integrator-operator/pull/25